std_msgs/Header header
string[] uuids             # human ids
sensor_msgs/Image image     # image when human ids were tracked
